connector-splunk/src/main/resources/application.properties

```diff
@@ -36,3 +36,9 @@ mp.messaging.tocamel.topic=platform.notifications.tocamel
 mp.messaging.fromcamel.topic=platform.notifications.fromcamel
 
 %test.quarkus.log.category."com.redhat.cloud.notifications".level=DEBUG
+
```

Comment:
Isn't it possible to move it into secrets-loader module ?

Reply:
The configuration below? No that's not possible.

Reply:
Ok